src/base/bittorrent/session.cpp

```diff
@@ -479,7 +479,7 @@ Session::Session(QObject *parent)
         m_storedCategories = map_cast(m_categories);
     }
 
-    m_tags = List::toSet(m_storedTags.get());
+    m_tags = {m_storedTags.get().cbegin(), m_storedTags.get().cend()};
```

Comment:
the `get()` returns an instance of the object, you should use a temp variable to hold it.

Reply:
Wouldn't it be better if `get()` returned a `const reference` instead?

Reply:
>Wouldn't it be better if get() returned a const reference instead?

Can't be sure, it might make the getter functions look weird.